apps/web/components/ProductProperties/ProductProperties.vue

```diff
@@ -1,50 +1,52 @@
 <template>
-  <div class="px-4" data-testid="product-properties">
-    <fieldset v-if="sizes?.length" class="pb-4">
-      <legend class="block mb-2 text-base font-medium leading-6 text-neutral-900">{{ $t('size') }}</legend>
-      <span v-for="{ label, value } in sizes" class="mr-2 mb-2 uppercase" :key="value">
-        <SfChip
-          class="min-w-[48px]"
-          size="sm"
-          :input-props="{
-            onClick: (e) => value == selectedSize && e.preventDefault(),
-          }"
-          :model-value="value === selectedSize"
-          @update:model-value="value !== selectedSize && setAttribute('size', value)"
-        >
-          {{ label }}
-        </SfChip>
-      </span>
-    </fieldset>
-    <fieldset v-if="sizes?.length" class="pb-2">
-      <legend class="block mb-2 text-base font-medium leading-6 text-neutral-900">{{ $t('color') }}</legend>
-      <span v-for="{ label, value } in colors" class="mr-2 mb-2 uppercase" :key="value">
-        <SfChip
-          class="min-w-[48px]"
-          size="sm"
-          :input-props="{
-            onClick: (e) => value == selectedSize && e.preventDefault(),
-          }"
-          :model-value="value === selectedColor"
-          @update:model-value="setAttribute('color', value)"
-        >
-          <template #prefix><SfThumbnail size="sm" :style="{ background: value }" /></template>
-          {{ label }}
-        </SfChip>
-      </span>
-    </fieldset>
+  <div v-if="attributeGroups && attributeGroups.length" class="px-4" data-testid="product-properties">
+    <SfSelect class="mb-2" v-for="(attributeGroup, index) in attributeGroups" :key="index"
+      @update:modelValue="changeVariationId" v-model="selectedVariation" size="sm" placeholder="-- Select --">
+      <option v-for="{ value, label } in attributeGroup.map(x => ({ value: x.variationId.toString(), label: x.value }))" :key="value"
```

Comment:
I think it would be better for complex logic to be placed in the script, not in the template.